From 8d6df83c23414538e4601f810ab0800a4b80a2d2 Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Tue, 9 Oct 2018 11:41:38 +0200
Subject: [PATCH 04/10] ptxd_make_xpkg_finish_impl: propagate errors properly

Otherwise opkg-build errors for the real package will be lost.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 scripts/lib/ptxd_make_xpkg_finish.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/lib/ptxd_make_xpkg_finish.sh b/scripts/lib/ptxd_make_xpkg_finish.sh
index 3b1bd61..9bef818 100644
--- a/scripts/lib/ptxd_make_xpkg_finish.sh
+++ b/scripts/lib/ptxd_make_xpkg_finish.sh
@@ -15,7 +15,7 @@
 ptxd_make_xpkg_finish_impl() {
     chown -R 0:0 "${pkg_xpkg_tmp}" "${pkg_xpkg_dbg_tmp}" &&
     ptxd_make_xpkg_pkg "${pkg_xpkg_tmp}" "${pkg_xpkg_dbg_tmp}" "${pkg_xpkg_cmds}" "${pkg_xpkg_perms}" &&
-    opkg-build ${ptx_xpkg_extra_args} "${pkg_xpkg_tmp}" "${ptx_pkg_dir}"
+    opkg-build ${ptx_xpkg_extra_args} "${pkg_xpkg_tmp}" "${ptx_pkg_dir}" &&
     if [ "$(find "${pkg_xpkg_dbg_tmp}" -type f | wc -l)" -gt 1 ]; then
 	# more than just the control file
 	opkg-build ${ptx_xpkg_extra_args} "${pkg_xpkg_dbg_tmp}" "${ptx_pkg_dir}"
-- 
2.7.4

